sections/07_conclusions.md

```diff
@@ -91,22 +91,41 @@ very large collections of input data into interpretable models that spur
 scientists to ask questions that they didn't know how to ask, it will be clear
 that deep learning has transformed biology and medicine.
 
+### Continuous collaborative manuscript drafting
```

Comment:
The transition from the conclusions to this section feels abrupt.  We close with our big punchline at the end of the conclusion and then move to this methods-like section.

Would moving it below the author contributions or acknowledgements help signal that we've moved to a new section of the manuscript?  I'm not sure of any better places to put it.